controllers/component/component_controller.go

```diff
@@ -61,11 +61,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	component := &applicationapiv1alpha1.Component{}
 	err := r.Get(ctx, req.NamespacedName, component)
 	if err != nil {
-		logger.Error(err, "Failed to get component for", "req", req.NamespacedName)
```

Comment:
I think that these changes should go into the [buildpipeline controller](https://github.com/redhat-appstudio/integration-service/blob/main/controllers/buildpipeline/buildpipeline_controller.go) because STONEINTG-645 is geared towards the missing component in the build pipeline data.

Reply:
I thought putting them in the component controller seemed strange - guess I should have voiced that concern.  That said, the ticket says "this error that's shown below is the expected state, because not every build pipeline on cluster belongs to RHTAP so they don't have a component associated with them via the component label."  It seems like the change should go [here](https://github.com/redhat-appstudio/integration-service/blob/41b7f7cdd0c5503ca5d013bad317ef09fa19c378/controllers/buildpipeline/buildpipeline_controller.go#L82-L87), correct?

Reply:
Yup, that should be the place for it.

Reply:
@MartinBasti Since you created the original ticket, do you have any objections?

Reply:
no objections, I agree, ticket is about build pipeline